packages/react-instantsearch/src/widgets/Panel.js

```diff
@@ -0,0 +1,37 @@
+import Panel from '../components/Panel.js';
+
+/**
+ * The Panel component will add a title to a widget. It will also handle when a wrapped widget doesn't have any refinement:
```

Comment:
Proposal:

```txt
The Panel widget wraps other widgets with a consistent panel design. It also reacts, indicates and set CSS classes when widgets are no more relevant for refining (like when a RefinementList becomes empty because of the current search).
```

I won't list the widgets, ultimately Panel can be used with any widget right?